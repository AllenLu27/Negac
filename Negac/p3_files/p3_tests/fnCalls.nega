int f1(){
}
void f2(){
}
void f3(int a,bool b){
}
bool f4(){
	f1();
	f2();
	f3(1,2);
}
