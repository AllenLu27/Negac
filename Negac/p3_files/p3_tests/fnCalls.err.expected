FATAL [10,7]: Type of actual does not match type of formal
Semantic analysis failed
